src/linpred.jl

```diff
@@ -116,6 +116,8 @@ Base.cholfact!{T}(p::SparsePredChol{T}) = p.chol
 
 invchol(x::DensePred) = inv(cholfact!(x))
 invchol(x::SparsePredChol) = cholfact!(x) \ eye(size(x.X, 2))
+# TODO: support fweights, pweights, and aweights
+# This vcov represents σ^2  X'X
 vcov(x::LinPredModel) = scale!(invchol(x.pp), dispersion(x, true))
```

Comment:
Need to check dispersion(x, true) will use nobs, which will use the right behavior for fweights vs pweights